modules/core/src/v2/coins/stx.ts

```diff
@@ -15,6 +24,35 @@ interface SupplementGenerateWalletOptions {
   rootPrivateKey?: string;
 }
 
+export interface TransactionFee {
+  fee: string;
+}
+export interface StxTransactionExplanation extends TransactionExplanation {
+  memo?: string;
+  type?: number;
+  contractAddress?: string;
+  contractName?: string;
+  contractFunction?: string;
+  contractFunctionArgs?: { type: string; value: string }[];
+}
+
+export interface ExplainTransactionOptions {
+  txHex?: string; // txHex is poorly named here; it is just a wrapped JSON object
+  halfSigned?: {
+    txHex: string; // txHex is poorly named here; it is just a wrapped JSON object
```

Comment:
rename: txJson?

Reply:
The comments are from the code copied from the `trx.ts` coin.  It's not accurate -- the `txHex` field really is a hex string, not JSON.  I'll remove the comments.